test/common/stats/allocator_impl_test.cc

```diff
@@ -139,6 +139,16 @@ TEST_F(AllocatorImplTest, RefCountDecAllocRaceSynchronized) {
   EXPECT_FALSE(alloc_.isMutexLockedForTest());
 }
 
+TEST_F(AllocatorImplTest, HiddenGauge) {
+  GaugeSharedPtr hidden_gauge =
+      alloc_.makeGauge(makeStat("hidden"), StatName(), {}, Gauge::ImportMode::HiddenAccumulate);
+  EXPECT_EQ(hidden_gauge->importMode(), Gauge::ImportMode::HiddenAccumulate);
+
+  GaugeSharedPtr non_hidden_gauge =
+      alloc_.makeGauge(makeStat("non_hidden"), StatName(), {}, Gauge::ImportMode::Accumulate);
```

Comment:
add `EXPECT_FALSE(non_hidden_gauge->hidden());` and add a third case for Gauge::ImportMode::NeverImport to check it's also not hidden.

Reply:
Done